Source/AliveLib/Glukkon.cpp

```diff
@@ -2565,9 +2565,90 @@ void Glukkon::GetOnPlatforms_444060()
         (TCollisionCallBack)&BaseAliveGameObject::OnTrapDoorIntersection_408BA0);
 }
 
-void CC Glukkon::PlaySound_4447D0(int /*sndIdx*/, Glukkon* /*pGlukkon*/)
+SfxDefinition stepSfx_554840[3] =
 {
-    NOT_IMPLEMENTED();
+    { 0u, 8u, 36u, 25u, 1524, 1905 },
+    { 0u, 3u, 59u, 60u, 0, 254 },
+    { 0u, 3u, 72u, 120u, 0, 254 }
+};
+
+void CC Glukkon::PlaySound_4447D0(int sndIdx, Glukkon* pGlukkon)
+{
+    int volumeLeft, volumeRight;
+    auto pGlukkonBase = static_cast<BaseAnimatedWithPhysicsGameObject*>(pGlukkon);

```

Comment:
ditto - redundant cast, no need to cast to base types, like polymorphism do its thing